.github/workflows/validate-pr.yaml

```diff
@@ -129,3 +129,61 @@ jobs:
       - name: Run npm format
         run: |
           cd web && npm run format
+  run-k8s-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Start minikube
+        uses: meroxa/setup-minikube@master
+        with:
+          minikube-version: 1.24.0
+      - name: run k8s deployment on the cluster
+        run: |
+          namespace=monocle
+          kubectl create namespace "${namespace}" 
+          kubectl create --namespace "${namespace}" --filename ./deployment/ 
+
+          deploy_pod_labels_raw=$(kubectl get --namespace "${namespace}" --filename ./deployment/elastic-deployment.yaml --output go-template='{{ range $k, $v :=.spec.selector.matchLabels }}{{$k}}={{$v}},{{ end }}')
+          deploy_pod_labels_elastic=$(echo "${deploy_pod_labels_raw}" | rev | cut --delimiter=',' --fields='2-' | rev)
+
+          while [[ $(kubectl get po --namespace "${namespace}" --no-headers 2>/dev/null | wc -l) -eq  0 ]]; do
+            echo -n "."
+            sleep 1s
+          done
+          echo
+
+          kubectl wait --for=condition=Ready --namespace "${namespace}" pods --selector "${deploy_pod_labels_elastic}" --timeout=1m
+          kubectl wait --for=condition=Ready --namespace "${namespace}" pods --all
+
+          kubectl create --namespace "${namespace}" job test  --from $( kubectl get --namespace ${namespace} -f ./deployment/crawler-legacy-cronjob.yaml --output name) 
+          kubectl wait --for=condition=complete --namespace "${namespace}" job/test 
+
+      - name: Display indexes
+        run: |
+          namespace=monocle
+          kubectl port-forward --namespace "${namespace}" $( kubectl get --namespace "${namespace}" -f deployment/elastic-service.yaml --output name) 9200:9200 &
+          sleep 5s
```

Comment:
nit: It seems curl have a "--retry" option

Reply:
I tried using the retry, but 404 is not recoverable (won't retry) @morucci

Reply:
> @TristanCacqueray @morucci New mergify permissions https://github.com/change-metrics/monocle/pull/732/checks?check_run_id=4417182506

Ah seem it is : https://github.com/change-metrics/monocle/pull/773